tests/manage/z_cluster/cluster_expansion/conftest.py

```diff
@@ -69,23 +69,35 @@ def factory(ocs_nodes=False, node_count=3, taint_label=None):
             log.info(
                 f"The worker nodes number before expansion {len(get_worker_nodes())}"
             )
-            if config.ENV_DATA.get("rhel_user"):
-                node_type = constants.RHEL_OS
+            dt = config.ENV_DATA["deployment_type"]
+            if dt == "ipi":
+                machines = machine_utils.get_machinesets()
+                log.info(
```

Comment:
I think we don't need this line here since we have it in line 69 https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/z_cluster/cluster_expansion/conftest.py#L69

Reply:
ack got it